src/deploy.ts

```diff
@@ -63,7 +63,7 @@ async function run() {
   await ux.print(`📦 Deploying ${ux.colors.white(STACK_REPO)}:${ux.colors.white(STACK_TAG)} to ${ux.colors.green(STACK_ENV)} cluster`)
   console.log('\n')
 
-  await exec(`./node_modules/.bin/cdk deploy ${STACKS[STACK_ENV].join(' ')} --outputs-file outputs.json`, {
+  await exec(`npx aws-cdk deploy ${STACKS[STACK_ENV].join(' ')} --outputs-file outputs.json`, {
```

Comment:
@klebertertulino Are we sure we want to do this? Is there a possibility that npx might run a different version of the cdk cli then what we have installedin the package? Running from local bin ensures version compatibility.